### ISSUE TITLE
'FirebaseDatabaseUI/FirebaseDatabaseUI.h' file not found

### ISSUE THREAD

**Issue by fabyeah** [oijusti/react-native-firebaseui-auth#55]
Seems to be similar to #38 

<img width="814" alt="Screenshot 2023-03-30 at 06 21 05" src="https://user-images.githubusercontent.com/4738527/228729495-8582c02a-f6e4-4d37-a911-f0e5b110aed4.png">

In Podfile I have

`use_frameworks! :linkage => :static`

`use_modular_headers!`

`pod 'FirebaseUI'`

**Comment by fabyeah** [comment]
The file does exist at that location though:

<img width="271" alt="Screenshot 2023-03-30 at 22 26 17" src="https://user-images.githubusercontent.com/4738527/228956402-c66112a4-b30e-4104-8935-1cd2297f6419.png">

**Comment by hunglmtb** [comment]
It seems to be the newer version of FirebaseUI at dependencies. I tried to add these to `react-native-firebaseui-auth.podspec `and it works now
  ```
s.dependency 'FirebaseDatabaseUI'
  s.dependency 'FirebaseAuthUI'
  s.dependency 'FirebaseFirestoreUI'
  s.dependency 'FirebaseOAuthUI'
  s.dependency 'FirebasePhoneAuthUI'
  s.dependency 'FirebaseAnonymousAuthUI'
  s.dependency 'FirebaseEmailAuthUI'
  s.dependency 'FirebaseFacebookAuthUI'
  s.dependency 'FirebaseGoogleAuthUI'
  s.dependency 'FirebaseStorageUI'
```

@fabyeah You can refer my folk commit at
https://github.com/hunglmtb/react-native-firebaseui-auth/commit/594ed7a1d880d410525cd8ee89680e89b7593db4

**Comment by oijusti** [comment]
Thanks @hunglmtb! 

@fabyeah this issue has been fixed, please upgrade to version 1.4.0 that supports iOS FirebaseUI 13
